tools/topology/topology2/production/tplg-targets-ace1.cmake

```diff
@@ -132,6 +132,14 @@ BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=7,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true,\
 GOOGLE_RTC_AEC_SUPPORT=1,DEEP_BUF_SPK=true,PLAYBACK_PIPELINE_SRC=dts,\
 SSP_SPK_FMT_24=true,SSP_HS_FMT_24=true"
 
+"cavs-rt5682\;sof-mtl-rt5650-google-aec\;PLATFORM=mtl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,\
+PDM1_MIC_B_ENABLE=1,DMIC0_PCM_ID=99,PREPROCESS_PLUGINS=nhlt,\
+NHLT_BIN=nhlt-sof-mtl-rt5650-google-aec.bin,DEEPBUFFER_FW_DMA_MS=10,HEADSET_SSP_DAI_INDEX=2,\
+SPK_ID=6,SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,SPEAKER_CODEC_NAME=SSP0-Codec,\
+BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=7,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true,\
+GOOGLE_RTC_AEC_SUPPORT=1,DEEP_BUF_SPK=true,PLAYBACK_PIPELINE_SRC=volume,\
+SSP_SPK_FMT_24=true,SSP_HS_FMT_24=true"
+
```

Comment:
is this really a 'production' topology? I am starting to think a lot of those topologies are for test and development, and don't need to be handled in this folder.

Reply:
Ack,, @andyross @cujomalainey that's a good question. The topologies in production are going to be shipped via sof-bin and e.g. included verbatim to many Linux distributions. Rationale is to ship all topologies we refer to in upstream Linux kernel. There are some exceptions due to historical reasons, but target is to distribute all the files in production folder. This is Intel only now, but all vendors are really welcome to add their binaries to the sof-bin releases and we are keeping the process that way on purpose. 

If you have topologies that are only for development, we have a separate "topology2/development" folder.

Reply:
Is the suggestion to move it to ../development/tplg-targets.cmake?  I can do that.  (Though there's some practical friction there as `scripts/build-tools.sh -T` won't build non-production topologies by default, meaning you can't have a "test" that uses the same build scripting as "production").

I guess the counterargument is maintenance: this is all boilerplate, and you want repeated boilerplate to live next to its "siblings" where possible so that when someone comes by to change something they don't miss one.  IMHO "production" and "development" are the wrong axes here and we want to be grouping by platform.  Which I guess we are, as this is an ace file.

But I'll move it if you want.

Reply:
I'm neutral on this, just that if if it's under "production/", it's going to be shipped out.

Plain "scripts/build-tools.sh" will build the development topologies as well. Not sure why the "-T" has been special cased, but that indeed is the case.

Reply:
Maybe a better scheme would be to keep the topology descriptors (or whatever the cmake entries are called) in a hierarchy that reflects their commonality and keep a separate registry of "topologies to ship to sof-bin" by name somewhere else?

Reply:
@andyross we can move it back if we start shipping it, which is quite possible in the future